.github/workflows/performance_harness_run.yaml

```diff
@@ -2,6 +2,16 @@ name: "Performance Harness Run"
 
 on:
   workflow_dispatch:
+    inputs:
+      platform-choice:
+        description: 'Override build platform'
```

Comment:
I guess this isn't really overriding the platform but rather picking which platform